frontend/resources/banner.txt

```diff
@@ -1,6 +0,0 @@
-  ______________________________
- /  _____/\_   _____/\_   _____/
-/   \  ___ |    __)_  |    __)  
-\    \_\  \|        \ |     \   
- \______  /_______  / \___  /   
-        \/        \/      \/    The GEF
```

Comment:
You removed the banner we never used. However, we may want to use it in the future